### DISCUSSION TITLE
pycaret- remove perfect collinearity

### DISCUSSION THREAD

**Discussion by aartialight** [pycaret/pycaret#1486] [Q&A]
Hi, 
I have started working with pycaret, came across this feature in setup- remove perfect collinearity, default is true, which remove features that are 100% correlated. Can anyone please let me know what function it is using at the backend?

**Comment by moezali1** [comment]
@aartialight `remove_perfect_collinearity` is checking the Pearson correlation of all X_features and for perfectly correlated features, it will drop one of the two columns (priority is to drop column with lesser correlation with target)
